guide/sequelize/README.md

```diff
@@ -123,30 +123,30 @@ const Tags = sequelize.define('tags', {
 
 `type` refers to what kind of data this attribute should hold. The most common types are number, string, and date, but there are other data types that are available depending on the database.  
-`unique: true` will ensure that this field will never have duplicated entries. We'll disallow the existence of duplicate tag names in our database.  
-`defaultValue` lets us set a fallback value if we don't assign the value during the insert.  
+`unique: true` will ensure that this field will never have duplicated entries. Duplicate tag names will be disalowed in this database.  
+`defaultValue` allows to set a fallback value if no value is set during the insert.  
 `allowNull` is not all that important, but this will guarantee in the database that the attribute is never unset. You could potentially set it to be a blank or empty string, but has to be set to _something_.
 
 ::: tip
-`Sequelize.STRING` vs `Sequelize.TEXT`: In most database systems, the length of the string is a fixed length for performance reasons. Sequelize defaults this to 255. Use STRING if your input has a max length, and use TEXT if doesn't. For sqlite, there's no unbounded string type so it won't matter which one you pick.
+`Sequelize.STRING` vs `Sequelize.TEXT`: In most database systems, the length of the string is a fixed length for performance reasons. Sequelize defaults this to 255. Use STRING if your input has a max length, and use TEXT if does not. For sqlite, there is no unbounded string type so it will not matter which one you pick.
 :::
 
 ### [gamma] Syncing the model
 
-Now that we have defined our structure, we need to make sure the model exists in the database. This goes in our `.once('ready')` event. This way the table structure gets created, and we don't need to worry about it later.
+Now that our structure is defined, you need to make sure the model exists in the database. This goes in our `.once('ready')` event. This way the table structure gets created, and you do not need to worry about it later.
 ```js
 Tags.sync();
 ```
 
-The table doesn't actually get created until you `sync` it. The schema that we defined from before was simply creating the model that lets Sequelize know what our data should look like. For testing, you can use `Tags.sync({ force: true })` to recreate the table every time on startup. This way you can get blank slate each time.
+The table does not actually get created until you `sync` it. The schema that was defined from before was simply creating the model that lets Sequelize know what the data should look like. For testing, you can use `Tags.sync({ force: true })` to recreate the table every time on startup. This way you can get blank slate each time.
 
 ### [delta] Adding a tag
 
-We can finally get our first command. We'll start off with adding a tag.
+You can finally get your first command. Start off with adding a tag.
```

Comment:
We is fine here, as we - the creators of the guide - decide to start with this command
Getting a command sounds rather confusing, giving the sentence a bit more flesh can help here in my opinion:
```suggestion
After all this preparation you can now write your first command. We will start off with the ability to add a tag.
```